crates/runtime/src/mmap.rs

```diff
@@ -438,8 +442,23 @@ impl Mmap {
         #[cfg(not(windows))]
         {
             use rustix::mm::{mprotect, MprotectFlags};
+
+            if enable_branch_protection {
+                #[cfg(all(target_arch = "aarch64", target_os = "linux"))]
+                if std::arch::is_aarch64_feature_detected!("bti") {
+                    let prot = libc::PROT_EXEC | libc::PROT_READ | 0x10; // PROT_BTI
```

Comment:
Could this perhaps still fall through to `rustix` below but use `MprotectFlags::from_u32` until there's a constant for `PROT_BTI`?